README.md

```diff
@@ -194,17 +194,23 @@ http://markevans.github.io/dragonfly/cache
 
 ### Localization
 
-Alchemy ships with one default English translation for the admin interface. If you want to use the admin interface in other languages please have a look at the [`alchemy_i18n` project](https://github.com/AlchemyCMS/alchemy_cms).
+Alchemy ships with one default English translation for the admin interface. If you want to use the admin interface in other languages please have a look at the [`alchemy_i18n` project](https://github.com/AlchemyCMS/alchemy_i18n).
 
 ## Upgrading
 
-The Alchemy team takes upgrades very seriously and tries to make them as smooth as we can.
-Therefore we have build in upgrade tasks, that try to automate as much as possible.
-
-That's why after updating the Alchemy gem you should **always run the upgrader**:
+We, the Alchemy team, take upgrades very seriously and we try to make them as smooth as possible.
+Therefore we have build an upgrade task, that tries to automate the upgrade procedure as much as possible.
 
+That's why after Alchemy gem has been updated, with explicit call to:
```

Comment:
after **the** Alchemy gem has been updated:

Reply:
done (see pm in slack)